internal/rest/receipt/receiptstore.go

```diff
@@ -174,7 +171,7 @@ func (r *receiptStore) writeReceipt(requestID string, receipt map[string]interfa
 		// Check if the reason is that there is a receipt already
 		existing, qErr := r.persistence.GetReceipt(requestID)
 		if qErr == nil && existing != nil {
-			log.Warnf("%s: exiting   receipt: %+v", requestID, *existing)
+			log.Warnf("%s: existing   receipt: %+v", requestID, *existing)
```

Comment:
Note a whitespace mismatch after the change

Reply:
fixed